build/.vsts-ci.yml

```diff
@@ -172,7 +164,7 @@ jobs:
       sudo -E ./jenkins/ubuntu_clang.sh
     displayName: 'Build'
   - script: |
-      cd cmake && sudo -E ../build_all/linux/run_tests.sh run_valgrind
+      cd cmake && sudo -E ../build_all/linux/run_tests.sh
```

Comment:
does this run valgrind tests though?

Reply:
oh wait that's right I separated the build and test run steps. This is just for the openssl workaround

Reply:
Yes. The previous run shows all SKUs ran valgrind correctly. The script was only adding the OpenSSL workaround. CMake generation is controlling Valgrind enable/disable.

![image](https://user-images.githubusercontent.com/8635911/155239514-63b14419-62c9-400a-961b-c44b85940ada.png)